src/main/java/stsjorbsmod/cards/CustomJorbsModCard.java

```diff
@@ -38,6 +37,9 @@
     public boolean upgradedMetaMagicNumber = false;
     public boolean isMetaMagicNumberModified = false;
 
+    public boolean exert = false;
```

Comment:
I would recommend using a SpireField on AbstractCard rather than a field on CustomJorbsModCard; the latter will cause problems with cards like `Shrapnel Bloom` that can Exert arbitrary other cards (including base game colorless cards)